code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -202,6 +202,8 @@
 					return
 				name = reject_bad_text(name)
 				while(count--)
+					if(!LAZYLEN(reagents.reagent_list))
```

Comment:
I couldn't find a dedicated proc for checking if the reagent datum actually contained reagents. If I missed it please point it out.

Reply:
`reagents.volume > 0`?

Reply:
Actually good point, didn't think of that.